__tests__/implementation.test.js

```diff
@@ -0,0 +1,37 @@
+const data = require("../db/data/test-data/index");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const request = require("supertest");
+
+beforeEach(() => {
+  return seed(data);
+});
+afterAll(() => {
+  return db.end();
+});
+
+describe("app", () => {
+  it("GET 404 if Endpoint does not exist", () => {
+    return request(app)
+      .get("/api/topicz")
+      .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toBe("Path Not Found");
+      });
+  });
+  describe("/api/topics", () => {
+    it("GET 200 sends an array to the client", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.topics.length).toBe(3);
+          body.topics.forEach((topic) => {
+            expect(topic).toHaveProperty("slug");
+            expect(topic).toHaveProperty("description");
```

Comment:
Good tests! Would be better if you can also check the value is a certain datatype. `expect(typeof topic.slug).toBe("string")` or `objectContaining`/`toMatchObject` can do the job